scripts/build/package.js

```diff
@@ -68,6 +70,8 @@ function minifyAndWrite(filepath, rawJavaScript) {
 }
 
 // 'main'
+const iconsBarrelMjs = [];
+const iconsBarrelJs = [];
```

Comment:
I guess technically "Barrel" is not the correct term anymore. That said, unless you have any suggestions for names I don't think this has to change (as I can't think of an alternative).

Reply:
Technically it works in the exact same way, so I'm fine with that name too.